pywbem/_exceptions.py

```diff
@@ -699,3 +701,50 @@ def __init__(self, message, conn_id=None):
         assert message is None or isinstance(message, six.string_types), \
             str(type(message))
         super(ModelError, self).__init__(message, conn_id=conn_id)
+
+
+class ListenerError(Exception):
+    """
+    Abstract base class for exceptions raised by the pywbem listener (i.e.
+    class :class:`~pywbem.WBEMListener).
+    """
+
+    def __init__(self, message):
+        """
+        Parameters:
+
+          message (:term:`string`): Error message (will be put into `args[0]`).
+
+        :ivar args: A tuple (message, ) set from the corresponding init
```

Comment:
This definition is new to me.

Reply:
It causes a section "Variables" to be created with the specified attributes. We use this already in a number of places in order to document attributes that we inherit from base classes but that are not documented there. For example, the "args" attribute of "Exception".
Look at https://pywbem.readthedocs.io/en/latest/client/exceptions.html#pywbem.ConnectionError for an example.